dotcom-rendering/src/lib/rootStyles.ts

```diff
@@ -46,4 +46,9 @@ export const rootStyles = (
 		background: ${sourcePalette.brandAlt[400]};
 		color: ${sourcePalette.neutral[7]};
 	}
+
+	.ad-slot-container {
+		/* prevent third-party code from breaking our layout */
+		overflow-x: hidden;
+	}
```

Comment:
@guardian/commercial-dev could you confirm that this won’t prevent fluid ads being as wide as necessary?